### PR TITLE
Refuelling Jerry Cans At Fuel Station

### PR DESCRIPTION
**Pull request by DuckyDoesCode**
Issue #475 

Hasn't been tested...

### CODE REVIEW

**Comment by Jawshy** [comment]
Received this on mission load and in local/client.rpt: 
```
16:02:13 Warning Message: Script core\items\fn_jerryEmptyRefuel.sqf not found
```

**Comment by baua1310** [comment]
the fault is in Functions.hpp:

class jerryEmptyRefuel {}
must be
class emptyJerryRefuel {}

File: Altis_Life.Altis/Functions.hpp [hunk 1]
```diff
@@ -254,6 +254,7 @@ class Life_Client_Core {
 		class defuseKit {};
 		class flashbang {};
 		class jerryRefuel {};
+		class jerryEmptyRefuel {};
```
Comment (tkcjesse) [inline Altis_Life.Altis/Functions.hpp]: Your script is named: fn_emptyJerryRefuel

You need to change one or the other to make them match.

**Comment by DuckyDoesCode** [comment]
Oh yeah will change that thanks

**Comment by DuckyDoesCode** [comment]
There we go fixed :)

**Comment by BoGuu** [comment]
@DuckyDoesCode 

Where is _displayName defined?

**Comment by DuckyDoesCode** [comment]
It isn't I need to remove that aswell thanks

**Comment by Jawshy** [comment]
@DuckyDoesCode, cannot join mission; receive "[mission completed](https://community.bistudio.com/wiki/BIS_fnc_endMission)". The following are written to the client/local.rpt file: 
```
16:51:15 Error in expression <"STR_ISTR_Jerry_NotLookingFuelStation"}
if(player distance _fuelPump > 15) exitW>
16:51:15   Error position: <if(player distance _fuelPump > 15) exitW>
16:51:15   Error Missing )
16:51:15 File mpmissions\__CUR_MP.Altis\core\items\fn_emptyJerryRefuel.sqf, line 19
16:51:15 Error in expression <"STR_ISTR_Jerry_NotLookingFuelStation"}
if(player distance _fuelPump > 15) exitW>
16:51:15   Error position: <if(player distance _fuelPump > 15) exitW>
16:51:15   Error Missing )
16:51:15 File mpmissions\__CUR_MP.Altis\core\items\fn_emptyJerryRefuel.sqf, line 19
```
and
```
16:51:46 "Variable: life_fnc_emptyjerryrefuel_meta is not allowed TYPE: CODE NS: MN"
16:51:47 "Variable: life_fnc_emptyjerryrefuel is not allowed TYPE: CODE NS: MN"
```

**Comment by DuckyDoesCode** [comment]
Yeah I should really try and test this in the future ok so I'll fix the first one when I get home but anyone got an idea what the second is? Never seen anything like it?

**Comment by Jawshy** [comment]
It requires white-listing in [Config_SpyGlass.hpp](https://github.com/ArmaLife/Framework/blob/master/Altis_Life.Altis/Config_SpyGlass.hpp) under allowedVariables.

**Comment by BoGuu** [comment]
@DuckyDoesCode 

Firstly, if the player interrupts his filling process, he loses his empty jerry can and doesn't gain another.
Secondly, you haven't added any checks for player distance during the filling process.
Lastly, how does somebody initiate the filling process?

**Comment by DuckyDoesCode** [comment]
Idk hadn't thought this through... I'm gunna close it now haven't got enough time

**Comment by sargken** [comment]
Well i hope you get it working if you need help @DuckyDoesCode let me know.

**Comment by DuckyDoesCode** [comment]
@Sargken I won't be carrying it on as I have no point firstly and secondly if I did I wouldn't need any help it was only because I did not have access to arma the past couple days

**Pull request closed without merging**